config/default.json

```diff
@@ -0,0 +1,5 @@
+{
+  "jwt": {
+    "expiresIn": "1y"
```

Comment:
Can you add comment in json file how to specify in seconds, days as well please?

Reply:
Can't add comments in the JSON file. Should we move to a different type?

Reply:
Can we just always take values in hours or days?